types/request/package.json

```diff
@@ -1,15 +1,15 @@
 {
     "private": true,
     "name": "@types/request",
-    "version": "2.48.9999",
+    "version": "2.49.9999",
```

Comment:
Do not change this version; nothing has changed in this PR to require a version bump. These versions are normally tied to what version of a library is typed.